src/main/java/com/rackspace/salus/resource_management/services/ResourceManagement.java

```diff
@@ -266,7 +266,7 @@ public void handleEnvoyAttach(AttachEvent attachEvent) {
         Optional<Resource> existing = getResource(tenantId, resourceId);
 
         if (existing.isPresent()) {
-            log.debug("Found existing resource related to envoy: {}", existing.toString());
+            log.debug("Found existing resource related to envoy: {}", existing.get().toString());
```

Comment:
or just

```suggestion
            log.debug("Found existing resource related to envoy: {}", existing.get());
```

since slf4j will invoke `toString` when the debug log level is active.